packages/material-ui/src/Popper/Popper.js

```diff
@@ -102,11 +113,10 @@ const Popper = React.forwardRef(function Popper(props, ref) {
       },
       // We could have been using a custom modifier like react-popper is doing.
       // But it seems this is the best public API for this use case.
-      onCreate: createChainedFunction(handlePopperUpdate, popperOptions.onCreate),
```

Comment:
I can't remember why it was added in #12085. Let's try without 🤷‍♂️

Reply:
In RTL you create the placement with `flipPlacement` so when you change placement in an RTL environment you only create a new PopperJS which wouldn't update the child prop. Since it's now immediately derived we no longer need this line.